packages/ui/src/utils/UiHelperUtils.ts

```diff
@@ -22,6 +22,10 @@ export const UiHelperUtil = {
     return newUrl.hostname
   },
 
+  getTruncateAddress(address: string, chars: number) {
```

Comment:
There is `truncateAddress` helper in core. For this component, but I agree that ui helpers might be more appropriate place. Could you move one from core into ui and apply it in account-view.